docs/myrocks-index.md

```diff
@@ -18,6 +18,29 @@ engines, if your workload uses fast storage, such as SSD:
 
 * Ensures better IO capacity
 
+!!! important
+
+    The latest MyRocks storage engine is based on recent RocksDB source code, with minor modifications applied by Percona.
+
+    To check the RocksDB version currently used by MyRocks, run the following command:
+    
+    ```{.bash data-prompt="$"}
+    $ head storage/rocksdb/rocksdb/HISTORY.md
+    ```
+    
+    ??? example "Expected output"
+
+        ```{.text .no-copy}
+        # Rocksdb Change Log
```

Comment:
This information doesn't need to be there. I'd remove them and use an ellipsis to show that there's text above the text we want to show.

Reply:
updated